features/booking/filter_data.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Filter booking data
+  In order to know how many bookings Pension Wise have made
+  The booking manager will need booking data extracted from the booking system
```

Comment:
This doesn't describe filtering data. I'm not sure why we're introducing new acceptance criteria?